components/atg/Terminal/TerminalComponents/DedupulicationTerminal.tsx

```diff
@@ -0,0 +1,188 @@
+import React, {
+  useMemo,
+  useEffect,
```

Comment:
what are these separate terminals for deduce, record etc

Reply:
We had to create seperate terminals because specific text-based loading was happening in side-bar which would not be possible for a single terminal. As we had to run TEST_GENERATE query , DEDUP query etc. So having single terminal would increase the complexity for it.

Reply:
will discuss this offline.